config/routes.rb

```diff
@@ -6,4 +6,6 @@
   get '/logout', as: :logout, to: 'sessions#destroy'
 
   get 'place', to: 'static#place'
+  get '/autocomplete', to: 'google/autocomplete#index'
+  get '/show', to: 'places#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.